docs/get-started/developers/index.md

```diff
@@ -47,9 +47,29 @@ Our design system libraries and the documentation website offer assets and guida
 
 ## About web components
 
-Web components are based on a set of web platform APIs that help to create reusable and encapsulated UI elements. Those standards consist of custom elements, shadow DOM, and HTML Templates.
```

Comment:
there as to be phrasing content between headings, iirc. checking with @adamjohnson

Reply:
Greg answered this [in Slack](https://redhat-internal.slack.com/archives/C05GTN4E82G/p1719930122638249), but:

✔️ This works:
```html
<h1>Heading 1</h1>
<h2>Heading 2</h2>
...content...
```

❌  This does not:
```html
<h2>Heading 2</h2>
<h2>Heading 2</h2>
```

❌  Nor this:
```html
<h2>Heading 2</h2>
...content...
<h4>Heading 4</h4>
```

Reply:
Thanks for clarifying 😊